### PR TITLE
Documentation for mipow smart bulb

### PR DESCRIPTION
**Pull request by amahlaka**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#4043

### CODE REVIEW

File: source/_components/light.mipow.markdown [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+title: "Mipow Color"
+description: "Instructions how to setup Mipow color BT le bulb within Home Assistant."
+date: 2015-10-25
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: mipow.png
```
Comment (Landrash) [inline source/_components/light.mipow.markdown]: *No applicable to this line but can't comment on the picture.* I would suggest chaining the logo to just the company logo if possible.

File: source/_components/light.mipow.markdown [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+ha_category: Light
+ha_iot_class: "Local Polling"
+featured: false
+ha_release: xx
+---
+
+The `Mipow color` support is integrated into Home Assistant as a light platform. Please note that this will only work on linux and requires a bluetooth adapter with LE support!
```
Comment (Landrash) [inline source/_components/light.mipow.markdown]: Linux  instead of Linux.

Suggest rewording this to
> Support for the `Mipow color` is integrated into Home Assistant as a light platform. Please note that this will only work on a Linux and requires Bluetooth LE support trough an USB adapter or a platform such as the Raspberry Pi 3 that has Bluetooth LE built in.

File: source/_components/light.mipow.markdown [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Example of bulbs:
+
+- [Mipow Bulb](https://www.amazon.com/Bluetooth-Controlled-Dimmable-Decorative-Christmas/dp/B014H9YRS0/ref=pd_sbs_107_2?ie=UTF8&psc=1&refRID=8VF91QP5SHY2SJ398RTW)
```
Comment (Landrash) [inline source/_components/light.mipow.markdown]: Change this to product links on http://www.playbulb.com/en/playbulb-smart.html ?

File: source/_components/light.mipow.markdown [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+
+The bulb is controlled normally using Playbulb X app on smartphones
+
+To enable mipow control add following to  `configuration.yaml` file:
```
Comment (Landrash) [inline source/_components/light.mipow.markdown]: Double space infront of \`configuration.yaml`.

File: source/_components/light.mipow.markdown [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+```yaml
+# Example configuration.yaml entry
+light:
+  - platform: mipow
+```
+
+Configuration variables:
+
+- **name** (*Required*): Name for the bulb
```
Comment (Landrash) [inline source/_components/light.mipow.markdown]: Most platforms has the name variable as an optional variable. I would suggest falling back to device name presented by the Bluetooth protocol or the mac address.

File: source/_components/light.mipow.markdown [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+- **serial** (*Required*): Bluetooth address of bulb (ex:00:00:00:00:00:00)
+
+
+### {% linkable_title Example configuration %}
+
+Will connect to specifed bulb using Bluetooth le:
```
Comment (Landrash) [inline source/_components/light.mipow.markdown]: Suggest change wording to
> This will connect to  and enable control of the specified bulb using Bluetooth LE.

File: source/_components/light.mipow.markdown [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+```yaml
+# Example configuration.yaml entry
+light:
+  platform: mipow
+  name: room
+  serial: '55:37:4B:12:AC:E6'
+```
+
+Available effects: Colorloop
+NOTE: 2 flash settings available, slow and little faster, you HAVE to supply either rgb_color or brightness together with flash or it wont work. also, while they are called long and short, neither one of them will stop until new setting is sent
```
Comment (Landrash) [inline source/_components/light.mipow.markdown]: This could use some rewording and a full example of how and where to use. Second sentence is also missing a capital letter.

File: source/_components/light.mipow.markdown [hunk 8]
```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+title: "Mipow Color"
+description: "Instructions how to setup Mipow color BT le bulb within Home Assistant."
```
Comment (Landrash) [inline source/_components/light.mipow.markdown]: Suggest you change this to
>"Instructions how to setup the Mipow color Bluetooth LE bulb within Home Assistant."

**Comment by Landrash** [comment]
Closing since parent pr has gone stale and has been closed.

**Comment by derfloDev** [comment]
Can someone please finish this component. I have two playbulbs and would love to add them to my home assistant setup. I already tried to program it by myself. Unfortunately it does not work so well.

**Comment by Landrash** [comment]
@derfloDev Have a look at the component thread instead since this PR is for the documentation.
